frontend/subrequests/lint/lint.go

```diff
@@ -39,6 +39,24 @@ type Warning struct {
 	Location    pb.Location `json:"location,omitempty"`
 }
 
+func (w *Warning) PrintToWriter(wr io.Writer, sources []*pb.SourceInfo) error {
```

Comment:
nit: `PrintTo` is enough for name (usually it is `WriteTo()` but in that case only one parameter is expected and written bytes count should be returned as well)